services/catalog/src/simcore_service_catalog/db/repositories/services.py

```diff
@@ -243,8 +244,8 @@ async def create_or_update_service(
         return created_service
 
     async def update_service(
-        self, patched_service: ServiceMetaDataAtDB
-    ) -> ServiceMetaDataAtDB:
+        self, patched_service: ServiceMetaDataAtDB, *, returning: bool = False
```

Comment:
Question: I always ask myself this. What is the point of having a default parameter if we control the whole code?
1. Are there a lot of locations where we use it and it's painful to change?
2. Does it change something in terms of performance to return the service after the update?